examples/eks-quickstart-app-dev.yaml

```diff
@@ -0,0 +1,27 @@
+# An example of ClusterConfig object compatible with the "app-dev" quickstart
+# profile. See also: https://github.com/weaveworks/eks-quickstart-app-dev
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-1
```

Comment:
Please follow the convention by calling this `cluster-12` and adding `12-` prefix in the filename.

Reply:
Ha, I had not realised this, apologies & thanks for pointing this out! 🙇